temporal-sdk/src/main/java/io/temporal/client/schedules/ScheduleClientOptions.java

```diff
@@ -106,30 +111,44 @@ public Builder setContextPropagators(List<ContextPropagator> contextPropagators)
       return this;
     }
 
+    /**
+     * Set the interceptors for this client.
+     *
+     * @param interceptors specifies the list of interceptors to use with the client.
+     */
+    public Builder setInterceptors(List<ScheduleClientInterceptor> interceptors) {
```

Comment:
Was going to comment that we need to update the `toString` etc, but `ScheduleClientOptions ` doesn't implement those. Will make a separate issue to address this